niimpy/location.py

```diff
@@ -147,18 +190,54 @@ def compute_distance_features(df):
             dists[i] = geodesic(loc1, loc2).meters
 
         speeds = dists / time_deltas
+        speeds[0] = 0
         speed_average = np.nanmean(speeds)
         speed_variance = np.nanvar(speeds)
         speed_max = np.nanmax(speeds)
-
         total_dist = sum(dists)
 
+        static_bins = speeds < 0.277
```

Comment:
Is 0.277 a threshold? If it is then it should not be hardcoded. Otherwise, it should be defined as a constant.